docs/CHANGELOG.rst

```diff
@@ -19,6 +19,8 @@ Changed
 
 Fixed
 -----
+- Report gene body coverage by ``rnaseqc-qc`` and assigned reads
```

Comment:
With this kind of wording, this could also be considered a change, not a fix. If you move it to Changed, this should also be reflected in the version number (1.20.1 -> 1.21.0).